packages/terra-dropdown-button/src/DropdownButton.jsx

```diff
@@ -127,6 +135,18 @@ class DropdownButton extends React.Component {
     return this.buttonNode;
   }
 
+  getSelectedOptionText = (selectedOptionText) => {
+    this.setState({ selectText: selectedOptionText });
+  }
+
+  handleFocus = () => {
+    this.setState({ selectText: '' });
+  };
+
+  handleBlur = () => {
+    this.setState({ selectText: '' });
```

Comment:
why do we need selectText reset on dropdown button focus ..??

Reply:
When we click on the button and select list-item it will announce the selected state and again if we revisit it is maintaining the aria-label value in the state,so to avoid this we are resetting the aria-label value on onFocus.

Reply:
I think then `onBlur` would help to take care of that. `onFocus` is not required for button I believe.

Reply:
yes, onBlur is satisfying the requirement so removed onFocus.